<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>79</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="ProductFrame">
     <property name="styleSheet">
      <string notr="true">QFrame#ProductFrame{
	background-color: #ffff00;
	border-radius: 2px;
	border-color: #000000;
	border-width: 2px;
	border-style: solid;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0,0,0,0,0">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="ProductNameLabel">
                  <property name="font">
                   <font>
                    <family>Futura</family>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QLabel {
    background-color:#ffff00;
	color: #000000;
}</string>
                  </property>
                  <property name="text">
                   <string>Product Name</string>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_6">
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="decreasePushButton">
                <property name="minimumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Futura</family>
                  <pointsize>24</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
    color: #ffff00;
    background-color: #31363b;
    border-width: 1px;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color : #31363b;
	background-color:#ffff00;
    border-width: 1px;
    border-color: : #ffff00;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}

QPushButton:disabled{
    color: rgb(82, 96, 117);
    background-color: rgb(0, 0, 0);
    border-width: 1px;
    border-color: rgb(82, 96, 117);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
                </property>
                <property name="text">
                 <string>-</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="ProductQtyLabel">
                <property name="minimumSize">
                 <size>
                  <width>30</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Futura</family>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLabel {
    background-color:#ffff00;
	color: #000000;
}</string>
                </property>
                <property name="text">
                 <string>Qty</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="increasePushButton">
                <property name="minimumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Futura</family>
                  <pointsize>24</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
    color: #ffff00;
    background-color: #31363b;
    border-width: 1px;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color : #31363b;
	background-color:#ffff00;
    border-width: 1px;
    border-color: : #ffff00;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}

QPushButton:disabled{
    color: rgb(82, 96, 117);
    background-color: rgb(0, 0, 0);
    border-width: 1px;
    border-color: rgb(82, 96, 117);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
                </property>
                <property name="text">
                 <string>+</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="TotalLabel">
                <property name="minimumSize">
                 <size>
                  <width>75</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>75</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Futura</family>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLabel {
    background-color:#ffff00;
	color: #000000;
}</string>
                </property>
                <property name="text">
                 <string>Total</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="delProductLinePushButton">
                <property name="minimumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
    color: #ffff00;
    background-color: #31363b;
    border-width: 1px;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color : #31363b;
	background-color:#ffff00;
    border-width: 1px;
    border-color: : #ffff00;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}

QPushButton:disabled{
    color: rgb(82, 96, 117);
    background-color: rgb(0, 0, 0);
    border-width: 1px;
    border-color: rgb(82, 96, 117);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
                </property>
                <property name="text">
                 <string>DEL</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
